### PR TITLE
Fix script to delete outdated .vg.json and .svg

### PR DESCRIPTION
**Pull request by chanwutk**

### CODE REVIEW

File: site/docs/mark/errorbar.md [hunk 1]
```diff
@@ -36,59 +39,59 @@ An error bar's mark definition contain the following properties:
...
 
-
-1) __Error bars showing standard error__ is the default error bar in Vega-Lite. It can also be explicitly specified by setting `extent` to `"stderr"`. The length of lower and upper rules represent standard error. By default, the rule marks expand from the mean.
+1. **Error bars showing standard error** is the default error bar in Vega-Lite. It can also be explicitly specified by setting `extent` to `"stderr"`. The length of lower and upper rules represent standard error. By default, the rule marks expand from the mean.
 
 <div class="vl-example" data-name="layer_point_errorbar_2d_horizontal"></div>
 
-
-2) __Error bar showing standard deviation__ can be specified by setting `extent` to `"stdev"`. For this type of error bar, the length of lower and upper rules represent standard deviation. Like an error bar that shows Standard Error, the rule marks expand from the mean by default.
+2. **Error bar showing standard deviation** can be specified by setting `extent` to `"stdev"`. For this type of error bar, the length of lower and upper rules represent standard deviation. Like an error bar that shows Standard Error, the rule marks expand from the mean by default.
```
Comment (kanitw) [inline site/docs/mark/errorbar.md]: Not sure what you're doing, but I like __ more than ** lol

Comment (chanwutk) [reply site/docs/mark/errorbar.md]: I think it is the prettier again...

But this branch is just for testing why github does not delete .svg, .vg.json, and normalized.vl.json that we no longer use.

`check-and-fix.sh` calls `yarn build:examples`. And, `yarn build:examples` calls `yarn build-examples.sh`, which runs:
```
rm -f examples/specs/normalized/*_normalized.vl.json
rm -f $dir/*.vg.json
```
So, I'm trying to see why this is not working.

**Comment by domoritz** [comment]
Try rebasing to master.

**Comment by chanwutk** [comment]
I noticed in `check-and-fix.sh`:
```
if [[ $node_version != '11' ]]; then
  exit 0;
fi
```
This may cause travis to exit since I see that travis use node 6. Could this be the reason for travis to exit before updating all the examples?

**Comment by domoritz** [comment]
No, we use node 11 on one build. See for example https://travis-ci.org/vega/vega-lite/builds/469134850

**Comment by domoritz** [comment]
Can you merge this into https://github.com/vega/vega-lite/pull/4311 so we can see whether it works?

**Pull request merged**